microcks.go

```diff
@@ -91,44 +97,76 @@ func WithSecondaryArtifact(artifactFilePath string) testcontainers.CustomizeRequ
 // WithArtifact provides paths to artifacts that will be imported within the Microcks container.
 // Once it will be started and healthy.
 func WithArtifact(artifactFilePath string, main bool) testcontainers.CustomizeRequestOption {
-	return func(req *testcontainers.GenericContainerRequest) {
+	return func(req *testcontainers.GenericContainerRequest) error {
 		hooks := testcontainers.ContainerLifecycleHooks{
 			PostReadies: []testcontainers.ContainerHook{
 				importArtifactHook(artifactFilePath, main),
 			},
 		}
 		req.LifecycleHooks = append(req.LifecycleHooks, hooks)
+
+		return nil
 	}
 }
 
-// WithNetwork allows to add a custom network
+// WithNetwork allows to add a custom network.
+// Deprecated: Use network.WithNetwork from testcontainers instead.
```

Comment:
Shouldn't we track this with a new issue? (just after this PR is merged)

Reply:
Yes, in fact!